build/apply-certs.sh

```diff
@@ -1,35 +1,42 @@
 #!/bin/bash
 
+if [ "$1" == "" ]; then
+  echo "usage: ./apply-certs.sh <cluster_context>"
+  exit 1
+fi
+
+
 set -e
 set -x
 
 # Paths to directories in which to store certificates and generated YAML files.
+CONTEXT="$1"
 DIR="$(pwd)"
 NAMESPACE="dss-main"
-CLIENTS_CERTS_DIR="$DIR/generated/$NAMESPACE/client_certs_dir"
-NODE_CERTS_DIR="$DIR/generated/$NAMESPACE/node_certs_dir"
-CA_KEY_DIR="$DIR/generated/$NAMESPACE/ca_key_dir"
-CA_CRT_DIR="$DIR/generated/$NAMESPACE/ca_certs_dir"
+CLIENTS_CERTS_DIR="$DIR/workdspace/$CONTEXT/client_certs_dir"
```

Comment:
Love this, I actually did something similar locally to handle multiple cluster

Reply:
+1 :)